kernel/common/process.rb

```diff
@@ -522,36 +522,39 @@ def self.coerce_rlimit_resource(resource)
 
   class Status
 
-    attr_reader :exitstatus
     attr_reader :termsig
     attr_reader :stopsig
 
-    def initialize(pid, exitstatus, termsig=nil, stopsig=nil)
+    def initialize(pid=nil, status=nil, termsig=nil, stopsig=nil)
       @pid = pid
-      @exitstatus = exitstatus
+      @status = status
       @termsig = termsig
       @stopsig = stopsig
     end
 
+    def exitstatus
+      return @status
```

Comment:
Good point, I'm so used to always using an explicit return that it goes automatically.